Corona-Warn-App/src/test/java/de/rki/coronawarnapp/eventregistration/events/server/qrcodepostertemplate/DefaultQrCodePosterTemplateSanityCheck.kt

```diff
@@ -0,0 +1,39 @@
+package de.rki.coronawarnapp.eventregistration.events.server.qrcodepostertemplate
+
+import android.content.Context
+import android.os.Build
+import androidx.test.core.app.ApplicationProvider
+import de.rki.coronawarnapp.util.HashExtensions.toSHA256
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import testhelpers.BaseTest
+import testhelpers.EmptyApplication
+
+@Config(sdk = [Build.VERSION_CODES.P], application = EmptyApplication::class)
+@RunWith(RobolectricTestRunner::class)
+class DefaultQrCodePosterTemplateSanityCheck : BaseTest() {
```

Comment:
Please add `*Test` in the name ?

Reply:
that's the pattern we use
- `DefaultAppConfigSanityCheck`
- `DefaultQrCodePosterTemplateSanityCheck`

I guess in that case it makes sense to rename both tests. What do you suggest?
`DefaultQrCodePosterTemplateSanityCheckTest` or `DefaultQrCodePosterTemplateTest` ?

Reply:
Whatever you like, I just need to distinguish them as `Test` classes

Reply:
renamed